packages/types/src/lib/interfaces.ts

```diff
@@ -1127,6 +1154,17 @@ export type AuthCallback = (params: AuthCallbackParams) => Promise<AuthSig>;
 /**
  * A map of node addresses to the session signature payload
  * for that node specifically.
+ * 
+ * Each individual session signature for each node includes the following properties:
+ * -  `sig`: The signature produced by the ed25519 key pair signing the `signedMessage` payload.
+ * 
+ * -  `derivedVia`: Should be `litSessionSignViaNacl`, specifies that the session signature object was created via the `NaCl` library.
+ * 
+ * -  `signedMessage`: The payload signed by the session key pair. This is the signed `AuthSig` with the contents of the AuthSig's `signedMessage` property being derived from the [`authNeededCallback`](https://v6-api-doc-lit-js-sdk.vercel.app/interfaces/types_src.GetSessionSigsProps.html#authNeededCallback) property.
+ * 
+ * -  `address`: The session key pair public key.
```

Comment:
@Ansonhkg @MaximusHaximus Is this actually a public key, or the derived address?

Reply:
In cases where the `session key` signs the `SIWE` it  will be the public key in its original format. But if it is an EOA wallet which signs then it will be the Ethereum address. It is a bit confusing as it can be two different data formats.